src/vs/workbench/contrib/chat/browser/media/chatSessions.css

```diff
@@ -235,20 +202,10 @@
 	margin: 0;
 }
 
-/* Show twistie only for collapsible items (like "Show history...") */
 .chat-sessions-tree-container .monaco-list-row[aria-expanded] .monaco-tl-twistie {
```

Comment:
This was really strange to me, seems like the treeview already supports hiding the twistie but it only removes the icon but doesn't change the padding.
Without this rule we get a really weird UI with lots of padding on the left